assets/app/view/game/bank.rb

```diff
@@ -43,7 +43,7 @@ def render_card
           h('div.title.nowrap', title_props, [h(:em, 'The Bank')]),
           h(:div, body_props, [
             h(:div, @game.format_currency(@game.bank.cash)),
-            h(GameInfo, game: @game, layout: 'discarded_trains'),
+            *h(GameInfo, game: @game, layout: 'discarded_trains'),
```

Comment:
render should always return one item, so remove the * and make sure GameInfo returns one div

Reply:
It already returns `''` if there are no discarded trains. I was avoiding the div-wrapper around header + table on purpose as these are exactly the ones I tried to eradicate. I’ll find another solution.